src/styles/Base/typography.scss

```diff
@@ -1,11 +1,43 @@
 /* Styles for fonts, line-height, paragraphs, h1, h2, h3 ... */
+html {
```

Comment:
Yes hte line bellow line 3 says `font-size: 1.6rem` 1 rem = 10px , 1.6 = 16px so thats our base size and by that we are going to do our typography :+1: